src/renderer/dx/DxFontInfo.cpp

```diff
@@ -198,7 +207,7 @@ void DxFontInfo::SetFromEngine(const std::wstring_view familyName,
     }
 
     THROW_HR_IF_NULL(E_FAIL, face);
-
+    _CalculateHashCode();

```

Comment:
This is I think the only meaningful place (inside `ResolveFontFaceWithFallback`) to add `_CalculateHashCode`.